Amazon.IonObjectMapper/Attribute/IonAnnotateTypeAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿/*
```

Comment:
So we're defining all the attributes in separate files? Might be worth consulting with @alpian to determine if we want to obey that rule or not?

Reply:
I would prefer to adhere to the StyleCop ruling for this PR.

The work was already done, and this can always be undone if we find it to be detrimental in the future.